components/tabs/tab.js

```diff
@@ -0,0 +1,47 @@
+import { css, html, LitElement, unsafeCSS } from 'lit';
+import { classMap } from 'lit/directives/class-map.js';
+import { getFocusPseudoClass } from '../../helpers/focus.js';
+import { RtlMixin } from '../../mixins/rtl/rtl-mixin.js';
+import { SkeletonMixin } from '../skeleton/skeleton-mixin.js';
+import { TabMixin } from './tab-mixin.js';
+
+class Tab extends TabMixin(SkeletonMixin(RtlMixin(LitElement))) {
+
+	static styles = css`
```

Comment:
This needs to pull in the mixin's styles, otherwise the wip tab component won't get any of those styles.
```javascript
static get styles() {
	const styles = [ css`
		...
	`];

	super.styles && styles.unshift(super.styles);
	return styles;
}
```

or 

```javascript
static get styles() {
	return [super.styles, css`
		...
	`];
}
```
The former approach being a bit more resilient in certain cases, but I think either will work in this case.

Reply:
Btw, we'll need to do the same thing in the mixin. Otherwise the skeleton styles won't get pulled properly. When pulling them in, applying the `skeleton` property should yield something like this:
![image](https://github.com/user-attachments/assets/9b12082d-816e-47f1-b2e3-4317e7816a92)